### DISCUSSION TITLE
핸드폰이 가로 모드일 때 cell 값이 변경되게 하고 싶습니다

### DISCUSSION THREAD

**Discussion by fenoteryn** [bosskmk/pluto_grid#153] [Q&A]
세로 모드와 가로 모드일 때의 cell 값을 다르게 하고 싶은데 세로 모드에서 가로 모드로 바뀌었을 때 column의 넓이도 그대로 유지됩니다. 이 부분을 어떻게 바꿔야 적용될까요?

**Comment by bosskmk** [comment]
~~컬럼의 넓이는 그리드를 생성 할 때 전달하는 컬럼속성에서 지정 할 수 있습니다.
그것을 동작 중에 바꾸기 위해서는 현재 사용자가 컬럼 우측 아이콘을 드래그 하는 것 뿐 입니다.
프로그래밍 적으로 변경하기 위해서는 기능 추가가 필요 할 거 같습니다.~~

예전에 resizeColumn 이라는 메서드를 추가 했었네요.
stateManager 의 resizeColumn 을 사용하시면 동적으로 변경 할 수 있습니다.
가로와 세로 모드 전환 되는 타이밍에 원하시는 값으로 변경 되도록 수정 하시면 될거 같습니다.
다만 지금은 컬럼 하나의 사이즈를 변경 할 수 있는 resizeColumn 이라서, 컬럼 여러개를 수정 하려면
함수를 그에 맞게 여러번 호출해야 합니다.
그렇게 되면 notifyListener 함수가 중복되어  호출 됩니다.
이 부분은 옵셔널 하게 변경을 해야 겠네요.

**Reply by fenoteryn** [reply]
감사합니다! 한 번 해 보겠습니다. 혹시 pre 버전에는 column 숨기기 기능이 없는 건가요?

**Reply by bosskmk** [reply]
column 숨기기는 1.1.0 부터 있습니다.
